pandas/io/parsers.py

```diff
@@ -358,10 +381,12 @@
 """
 
 _read_fwf_doc = """
-Read a table of fixed-width formatted lines into DataFrame
+Read a table of fixed-width formatted lines into DataFrame.
 
 %s
-""" % (_parser_params % (_fwf_widths, ''))
+""" % (_parser_params % (_fwf_widths, '',
```

Comment:
I think it would make more sense to have `{summary}` in the first line of `_parser_params` than this.

Sorry to be picky will all these, but I'd really want to simplify all this. The code of these functions is already very complex, having all this extra complexity for the docstrings makes things much worse IMO. I know it's from the original code, thanks for the help cleaning them.